api/operator_autopilot.go

```diff
@@ -352,22 +352,17 @@ func (op *Operator) AutopilotServerHealth(q *QueryOptions) (*OperatorHealthReply
 	r := op.c.newRequest("GET", "/v1/operator/autopilot/health")
 	r.setQueryOptions(q)
 
-	// we cannot just use requireOK because this endpoint might use a 429 status to indicate
-	// that unhealthiness
-	_, resp, err := op.c.doRequest(r)
+	// we use 429 status to indicate unhealthiness
+	d, resp, err := op.c.doRequest(r)
+	_, resp, err = requireHttpCodes(d, resp, err, []int{200, 429})
```

Comment:
Just to keep in line with other usages
```suggestion
	_, resp, err := requireHttpCodes(op.c.doRequest(r), []int{200, 429})
```

Reply:
Nvm! Discussed offline that this doesn't work as I thought (details [here](https://golang.org/ref/spec#Calls))